.codeclimate.yml

```diff
@@ -1,6 +1,32 @@
 version: "2"
 exclude_patterns:
 - rasa/core/utils.py   # codeclimate has some encoding issues with this files because of emojis
+- .*
+- .github/
+- CHANGELOG.mdx
+- CODEOWNERS
+- CODE_OF_CONDUCT.md
+- Dockerfile
+- LICENSE.txt
+- Makefile
+- NOTICE
+- PRONCIPLES.md
+- README.md
+- bandit.yml
+- binder/
+- changelog/
+- data/         # double-check this
+- docs/         # double-check this
+- examples/     # double-check this
```

Comment:
This is an interesting one because it's actual example bots so will have things like custom actions. I would say it's good to check